gnuradio-runtime/lib/buffer.cc

```diff
@@ -340,11 +340,20 @@ void buffer_reader::get_tags_in_range(std::vector<tag_t>& v,
 {
     gr::thread::scoped_lock guard(*mutex());
 
+    uint64_t lower_bound = abs_start - d_attr_delay;
+    // check for underflow and if so saturate at 0
+    if (lower_bound > abs_start)
+      lower_bound = 0;
```

Comment:
My wondering is whether the `lower_bound` should be 0 for either case. Need to follow the code for a bit to see how the methods `get_tags_*_bound` use the argument.

Reply:
You mean for the `else` case? Or for upper_bound?